packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -165,31 +94,65 @@ export const Datepicker: ComponentWithAs<'div', DatepickerProps> &
   const context = useFluentContext();
   const { setStart, setEnd } = useTelemetry(Datepicker.displayName, context.telemetry);
   setStart();
+  const dateSelectionErrorString = 'A date selection is required';
   const datepickerRef = React.useRef<HTMLElement>();
-  const [open, setOpen] = React.useState<boolean>(false);
+  const [openState, setOpenState] = useAutoControlled<OpenState>({
+    defaultValue: props.defaultCalendarOpenState ? OpenState.Open : OpenState.Closed,
+    value: undefined,
```

Comment:
```suggestion
    value: props.open,
```

Reply:
@layershifter  what would then be the difference between 'props.defaultOpen' and 'props.open'?